Functions/xASL_init_printSettings.m

```diff
@@ -23,7 +23,7 @@ function xASL_init_printSettings(x)
     %% Fallbacks
```

Comment:
Are these fallbacks the printed lines? Then I would make this clear in the variable name.
Also, the printed lines can be made a bit more reader friendly. `NII` = `NIfTI`, and `NIfTI` is actually `legacy` right? Why is `DEFACE` big caps and `Structural` not? The `NIfTI` phase should be fased out soon anyway, so I wouldn't care too much about that naming :)

Reply:
You mean NII in DCM2NII - this is not legacy, this is the middle step of DCM->NII->BIDS. Although the Legacy format is used there, we still need this step. In the future, we will probably try to do this smarter, but we will step have it.

My proposed solution:
- Put also `STRUCTURAL` and `POPULATION` in capitals.
- Keep NII.

Agreed?

Reply:
Done - do you agree?

Reply:
### Test

```
==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
Dataset Root        ...ATLAB\m.stritt\Server_xASL\New_TestData\Philips_PCASL_3DGRASE_volunteer
Import Modules      DCM2NII NII2BIDS DEFACE BIDS2LEGACY
Process Modules     STRUCTURAL ASL POPULATION 
bPause              False
iWorker             1
nWorkers            1
==============================================================================================
```

👍